cassandra-oss/Dockerfile.ubi7

```diff
@@ -0,0 +1,183 @@
+
+#############################################################
+# Build the Management API
+FROM --platform=$BUILDPLATFORM maven:3.6.3-jdk-8-slim as mgmtapi-setup
+
+WORKDIR /
+
+ENV MAAC_PATH /opt/management-api
+ENV DSE_HOME /opt/dse
+
+# copy over cassandra maven artifacts
+COPY --from=cass-builder /root/.m2 /root/.m2
+COPY pom.xml /tmp/pom.xml
+COPY management-api-agent-common/pom.xml /tmp/management-api-agent-common/pom.xml
+COPY management-api-agent-3.x/pom.xml /tmp/management-api-agent-3.x/pom.xml
+COPY management-api-agent-4.x/pom.xml /tmp/management-api-agent-4.x/pom.xml
+COPY management-api-agent-4.1.x/pom.xml /tmp/management-api-agent-4.1.x/pom.xml
+COPY management-api-agent-4.2.x/pom.xml /tmp/management-api-agent-4.2.x/pom.xml
+COPY management-api-common/pom.xml /tmp/management-api-common/pom.xml
+COPY management-api-server/pom.xml /tmp/management-api-server/pom.xml
+COPY settings.xml settings.xml /root/.m2/
+# this duplicates work done in the next steps, but this should provide
+# a solid cache layer that only gets reset on pom.xml changes
+RUN cd /tmp && mvn -q -ff -T 1C install -DskipOpenApi -P trunk && rm -rf target
+
+COPY management-api-agent-common /tmp/management-api-agent-common
+COPY management-api-agent-3.x /tmp/management-api-agent-3.x
+COPY management-api-agent-4.x /tmp/management-api-agent-4.x
+COPY management-api-agent-4.1.x /tmp/management-api-agent-4.1.x
+COPY management-api-agent-4.2.x /tmp/management-api-agent-4.2.x
+COPY management-api-common /tmp/management-api-common
+COPY management-api-server /tmp/management-api-server
+RUN mkdir $MAAC_PATH \
+    && cd /tmp \
+    && mvn -q -ff package -DskipTests -P trunk \
+    && find /tmp -type f -name "datastax-*.jar" -exec mv -t $MAAC_PATH -i '{}' + \
+    && rm $MAAC_PATH/datastax-mgmtapi-agent-3* \
+    && rm $MAAC_PATH/datastax-mgmtapi-agent-4.x* \
+    && rm $MAAC_PATH/datastax-mgmtapi-agent-4.1.x* \
+    && rm $MAAC_PATH/datastax-mgmtapi-*common* \
+    && cd ${MAAC_PATH} \
+    && ln -s datastax-mgmtapi-agent-4.2.x-0.1.0-SNAPSHOT.jar datastax-mgmtapi-agent-0.1.0-SNAPSHOT.jar \
+    && ln -s datastax-mgmtapi-agent-0.1.0-SNAPSHOT.jar datastax-mgmtapi-agent.jar \
+    && ln -s datastax-mgmtapi-server-0.1.0-SNAPSHOT.jar datastax-mgmtapi-server.jar
+
+###
+# Download CDC Agent jarfile
+###
+ARG CDC_AGENT_VERSION=2.2.2
+ARG CDC_AGENT_EDITION=agent-c4
+# Download CDC agent
+ENV CDC_AGENT_PATH=/opt/cdc_agent
+RUN mkdir ${CDC_AGENT_PATH} && \
+  curl -L -O "https://github.com/datastax/cdc-apache-cassandra/releases/download/v${CDC_AGENT_VERSION}/${CDC_AGENT_EDITION}-${CDC_AGENT_VERSION}-all.jar" && \
+  mv ${CDC_AGENT_EDITION}-${CDC_AGENT_VERSION}-all.jar ${CDC_AGENT_PATH}/cdc-agent.jar
+
+############################################################
+
+FROM registry.access.redhat.com/ubi7/ubi-minimal:${BASETAG} as cass-trunk
+
+LABEL maintainer="DataStax, Inc <info@datastax.com>"
+LABEL name="Apache Cassandra"
+LABEL vendor="DataStax, Inc"
+LABEL release="${VERSION}"
```

Comment:
`VERSION` isn't set anywhere, so this label will be empty. Either set it to `CASSANDRA_VERSION` from line 8 above, or dynamically figure out the version from the build process above